Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gem 'hydra', '~> 7.1.0'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 4.1.4'
+gem 'blacklight', github: 'projectblacklight/blacklight', branch: 'release-5.5'
```

Comment:
More...

In order use the local version of blacklight, you need to modify your bundler config file with:

```
bundle config --local local.blacklight /vagrant/vendor/gems/blacklight
```